ci/scripts/upload.sh

```diff
@@ -20,6 +20,7 @@ elif [[ "${BINARY_TYPE}" == "rpm" ]]; then
     bash upload.sh ksnip-*.rpm
 elif [[ "${BINARY_TYPE}" == "exe" ]]; then
     bash upload.sh ksnip-*.zip
+    bash upload.sh build/ksnip*.msi
```

Comment:
I believe the upload tool can upload multiple files at once, at least it seems to work for a whole directory.
Matching to the naming convention of other files it should be `ksnip-` I think.

Reply:
Yeah; i think it already has a ksnip- in the name ..